features/dd-sdk-android-profiling/src/main/java/com/datadog/android/profiling/Profiling.kt

```diff
@@ -38,8 +46,19 @@ object Profiling {
         val profilingFeature = ProfilingFeature(
             sdkCore = featureSdkCore,
             configuration = configuration,
-            profilerProvider = PerfettoProfilerProvider()
+            profiler = profiler
```

Comment:
by doing this it means we can have a possibility to receive profile result for the SDK core instance which didn't request it?

Also it seems by sharing `Profiler` we dispatch result only once, that doesn't work for the multiple SDK cores.